src/Components/Navbar.js

```diff
@@ -1,14 +1,49 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import { NavLink } from 'react-router-dom';
 import { toggleLogin } from '../Pages/LoginPage';
+import { toggleSignup } from '../Pages/SignupPage';
+import { logOutUser } from '../Redux/user/UserReducer';
+// import { toggleSignup } from '../Pages/SignupPage';
 import '../Styles/navbar.scss';
 
-export const toggle = () => {
+let navbarIsOpen = false;
+
+export const toggleNavbar = () => {
   const nav = document.getElementsByTagName('nav');
   nav[0].classList.toggle('invisible');
+  navbarIsOpen = !navbarIsOpen;
 };
 
 const Navbar = () => {
+  const user = useSelector((state) => state.user);
+  const dispatch = useDispatch();
+
+  const handleClick = () => {
+    dispatch(logOutUser());
+  };
+
+  console.log(handleClick);
```

Comment:
- Is this console output necessary?

Reply:
Oops! Nope! I'll comb through the consoles and take them out. Totally spaced on that. Good catch